portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/PageEvents.testcase

```diff
@@ -339,6 +358,25 @@ definition {
 	test CheckPageDepthReachedWhenPressPageDownKey {
 		property proxy.server.enabled = "true";
 
+		task ("Add a Blogs widget to page") {
+			ACUtils.addWidgetToPage(
+				layoutName = "AC Page",
+				widgetName = "Blogs");
+		}
+
+		task ("Add 10 blogs entries with long content") {
+			var n = "0";
+
+			while ("${n}" != "10") {
+				var n = ${n} + 1;
+
+				ACUtils.createBlogsAndAddToPage(
+					entryContent = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce in lacus nunc. Etiam et volutpat est. Quisque quis mi erat. Donec maximus mauris augue, nec convallis metus aliquet vitae. Nullam venenatis a nisl in tincidunt. Aliquam venenatis, enim in venenatis aliquam, mauris erat blandit nulla, sit amet tincidunt elit enim sit amet dolor. Aliquam fringilla condimentum varius. Maecenas tincidunt elit eget lacinia aliquam. Proin congue diam lorem. Vestibulum tristique elit mi, non dapibus erat lobortis.",
+					entryTitle = "Blogs Entry Title ${n}",
+					groupName = "Site Name");
+			}
```

Comment:
Hey @joshitagaki, great idea to add multiple blogs to increase page depth. I think you could use the `for` loop for that. It would be the same idea you had, but I think using `for` would be more beautiful and you wouldn't need that logic to increase the value of the variable. What do you think?